kvm/v12.3.0/announcement.md

```diff
@@ -0,0 +1 @@
+**Giant Swarm release v12.3.0 for KVM is available.** This release upgrades all Helm releases managed by Giant Swarm to use Helm 3. Further details can be found in the [release notes](https://github.com/giantswarm/releases/tree/master/KVM/v12.3.0).
```

Comment:
```suggestion
**Giant Swarm release v12.3.0 for KVM is available.** This release upgrades all Helm releases managed by Giant Swarm to use Helm 3. It also upgrades Container Linux to address security vulnerabilities. Further details can be found in the [release notes](https://github.com/giantswarm/releases/tree/master/KVM/v12.3.0).
```

@tfussell WDYT of including this here? Customers like knowing when security stuff is fixed right.